### ISSUE TITLE
Issue with Ensemble model.

### ISSUE THREAD

**Issue by MahsaSeifikar** [serengil/deepface#62]
When I want to use Ensemble model, like the tutorial"

`resp_obj = DeepFace.verify("img1.jpg", "img2.jpg", model_name = "Ensemble")`

I get the following error:
`There is an error with this image and modelCould not open /usr/local/lib/python3.6/dist-packages/deepface/models/face-recognition-ensemble-model.txt`

How we can solve that?

the library version is 0.0.31

**Comment by serengil** [comment]
It is interesting. face-recognition-ensemble-model.txt is in here: https://github.com/serengil/deepface/tree/master/deepface/models

Could you share what exist in /usr/local/lib/python3.6/dist-packages/deepface/

PS: If you copy the txt file I shared in the path specified in exception message, it will work.

**Comment by MahsaSeifikar** [comment]
Yeah, it's so strange! There is nothing in this path (.../deepface/models) except for __init__.py!

I used `pip install deepface`

**Comment by serengil** [comment]
Could you try to run the following command?

`pip install deepface==0.0.31 --ignore-installed`

**Comment by serengil** [comment]
Oke, I understand the problem. I will fix in 0.0.32 version. Until it is solved, you can copy the txt file manually.

**Comment by serengil** [comment]
new version is released. could you try it right now?

**Comment by MahsaSeifikar** [comment]
> new version is released. could you try it right now?

Unfortunately Not!
Another most strange thing is when I add this file on models directory and run my code both on Google Colab and on local my kernel dies!

**Comment by serengil** [comment]
deepface 0.0.32 comes with this txt file, you don't need to copy it anymore.

BTW, ensemble method requires lightgbm. Could you run the command "pip install lightgbm==2.3.1" before importing deepface.

**Comment by MahsaSeifikar** [comment]
It didn't solve my problem.
you can reach our my code in the following link. you can edit it if you want.

[https://colab.research.google.com/drive/1Vgn5FO52zz9awrQMtvtDUVJW9tRUwvvz?usp=sharing](Google Colab)


In the fourth cell, I got this error:
`LightGBMError: Could not open /usr/local/lib/python3.6/dist-packages/deepface/models/face-recognition-ensemble-model.txt`

**Comment by serengil** [comment]
Oke, that's a bug, I will resolve it soon.

**Comment by serengil** [comment]
I released version 0.0.33. Could you install this version and try again please?

**Comment by MahsaSeifikar** [comment]
It's work!!!! 
You save my day.
I am very excited to know what is the problem since I read your code and I did not notice what's the problem.

**Comment by serengil** [comment]
Let me explain the problem, maybe this helps some people having same trouble

Firstly, ensemble model needs .txt file and I stored in the repo: https://github.com/serengil/deepface/tree/master/deepface/models

Even though this text file exists in the repo, this won't be uploaded to PyPI. Before 0.0.32, this does not exist in the repo.

Then, I found that if you add this text file as script in [setup.py](https://github.com/serengil/deepface/blob/master/setup.py), then this will be uploaded to pypi. 

`scripts=['deepface/models/face-recognition-ensemble-model.txt'],`

Even though this is uploaded to pypi, this won't be copied when you call pip install command. That's why, I downloaded this text file externally in the code.

[DeepFace.py](https://github.com/serengil/deepface/blob/master/deepface/DeepFace.py)

```
home = str(Path.home())
				
if os.path.isfile(home+'/.deepface/weights/face-recognition-ensemble-model.txt') != True:
	print("face-recognition-ensemble-model.txt will be downloaded...")
	url = 'https://raw.githubusercontent.com/serengil/deepface/master/deepface/models/face-recognition-ensemble-model.txt'
	output = home+'/.deepface/weights/face-recognition-ensemble-model.txt'
	gdown.download(url, output, quiet=False)

ensemble_model_path = home+'/.deepface/weights/face-recognition-ensemble-model.txt'

deepface_ensemble = lgb.Booster(model_file = ensemble_model_path)
```

Funnily, this text file will be downloaded from GitHub repo :)
